.github/workflows/package-build.yml

```diff
@@ -66,18 +55,83 @@ jobs:
           asset_path: ${{ env.UPLOAD_FILE }}
           asset_name: ${{ env.UPLOAD_FILE }}
           asset_content_type: ${{ env.UPLOAD_ASSET_TYPE }}
+  build-windows:
+    runs-on: windows-2019
+    strategy:
+      fail-fast: false
+    steps:
+    - uses: actions/checkout@v3
+    #- name: msys setup
+    #  shell: 'C:/shells/msys2bash.cmd {0}'
+    #  run: |
+    #    set -euo pipefail
+    #    pwd
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3.11'
+    - name: build windows ouptuts
+      shell: cmd
+      run: |
+        pwd
+        call "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" x64
```

Comment:
Maybe make this path a variable?

Reply:
What do you mean? Assign it to an environment variable and then immediately using? I would understand the benefit of it, if it was used multiple times or the script was longer thus deserving configuration section at the start.

Reply:
I was thinking something like this: https://github.com/rizinorg/rizin/blob/dev/.appveyor.yml#L11
But if it's used only once maybe you are right and it doesn't worth extracting.